app/controllers/api/v2/json_web_tokens_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Api
+  module V2
+    class JsonWebTokensController < V2::BaseController
+      include Api::Version2
+
+      before_action :find_user
+
+      api :GET, "/users/:id/json_web_tokens", N_("Generate JSON web token")
```

Comment:
should this be POST?

Reply:
Well we do not create any new record in DB, just generate token  & return response, so I guess `GET` is correct. But no strong opinions here.

Reply:
GET should not have side effects so POST is correct IMHO.